tests/test_tocdirective/test_toc_latex_parts.tex

```diff
@@ -10,83 +10,41 @@
 
 \sphinxAtStartPar
 \sphinxstylestrong{Table of Contents}
-\begin{itemize}
```

Comment:
Note the change removes this weird lists that are created in LaTeX by `tableofcontents` directives.
To me though this looks like a bug, given that all the list items are empty.
If you still wanted this behaviour, then it should be added to jupyterbook-latex, since it is not a common output of sphinx

Reply:
cc: @AakashGfude can you review this?

Reply:
See also https://github.com/executablebooks/jupyterbook-latex/pull/55/files#r618795779

Reply:
@chrisjsewell the current jupyter-book created this list because `tableofcontents` directive logic was implemented here. Yes, ideally we should transfer it to `jupyterbook-latex` now.

Reply:
Well it seems correct to me that this list isn't present now, unless you know of a reason it should be?